README.md

```diff
@@ -1,9 +1,49 @@
-# Parity GitHub Action template
+# Get Fellows Action
 
-Template used to generate GitHub Actions.
+GitHub action to fetch all the fellows handles.
 
-## To start
+This action fetches all the fellows from the [on-chain data](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama-rpc.polkadot.io#/fellowship) and output two objects with the users’ GitHub information.
 
-- Remember to modify the `action.yml` file to have your required attributes and details. 
- - You can use [GitHub Action Brandings cheatsheet](https://github.com/haya14busa/github-action-brandings) to set the style of the action.
-- Remember to modify the name in the `package.json`.
+It looks in the fellows data for a field named `GitHub` and it extracts the handle from there.
+
+This action is intended to be chained with other actions that requires the fellows GitHub handles.
+
+## Installation
+This action is intended to be chained with other actions. It does not have any requirements.
+
+We simply need to add the step:
+```yaml
+- uses: paritytech/get-fellows-action
+  id: fellows
+```
+
+A working example where we print all the names is the following:
+```yaml
+name: Mention fellows
+
+on: [push]
+
+jobs:
+  mention-fellows:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: paritytech/get-fellows-action
+        id: fellows
+      - name: Mention them
+        run: |
+          echo "The fellows are $FELLOWS"
+        env:
+          # the handles of the fellows separated by commas
+          FELLOWS: ${{ steps.fellows.outputs.fellows }}"
+```
+
+### Outputs
+You can find all the outputs in the [`action.yml` file](./action.yml).
+
+They are:
+- `fellows`: All the fellows handles separated by commas
+	- Example: `user-1,user-2,user-3`
+- `fellows-handles`: A JSON array with all the fellows GitHub handles
+	- Example: `["user-1","user-2","user-3"]`
+- `fellows-map`: A JSON array with objects of type  "user": "github-handle", "rank": rank (as a number)
+	- Example: `[{"user":"user-1",rank: 4}, {"user":"user-2",rank: 2}, {"user":"user-3",rank: 1}]`
```

Comment:
it's currently mostly about mapping to GH handles, but it could potentially be used in more places and not only using github handles?  like mapping their other identity data for sending notifications like email, right?
so im thinking how 'user' would describe that it's a GH username?
may be `fellows-identity-map`? idk..

Reply:
I will rename this field to be github related so we can do as you suggested in the other comment. Good idea!